pom.xml

```diff
@@ -80,7 +80,7 @@ under the License.
     </licenses>
 
     <prerequisites>
-        <maven>3.0</maven> <!-- because this plugin is affected by: https://issues.apache.org/jira/browse/MNG-4814 and https://issues.apache.org/jira/browse/MSITE-495 -->
+        <maven>3.6.3</maven>
```

Comment:
Out of curiosity, is there a reason for requiring this specific version?

Reply:
this is the last stable version in 3.6 lineage which is 4+ years old
https://maven.apache.org/docs/history.html 
some plugins can be updated only when maven 3.6.3 is required
```
[INFO] Project defines minimum Maven version as: 3.6.3
[INFO] Plugins require minimum Maven version of: 3.6.3
[INFO] 
[INFO] No plugins require a newer version of Maven than specified by the pom.
[INFO] 
```
I would bump it to 3.9, but seems like default maven in GH runners is of lower version

Reply:
All good 👍